src/useTimeout/useTimeout.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useRef } from 'react'
```

Comment:
copy pasted from https://github.com/siddharthkp/use-timeout and tested

Reply:
We do we not simply use this package here?